src/modal.test.basics.ts

```diff
@@ -160,3 +160,19 @@ it('throws an error when the "tertiary" footer slot is the wrong type', async ()
     );
   });
 });
+
```

Comment:
Had to add these after adding [this](https://github.com/CrowdStrike/glide-core/pull/763/files#diff-81662b9e6278142471171b714e4db9d8d13ab7e21fee37d42384d94d96f34b5bR213-R219).